tools/packell/README.MD

```diff
@@ -22,7 +22,8 @@ To develop the Packell application, open `VS Code` at the root of the Packell pr
 
 ## Usage
 
-As part of nix setup, the `Packell` application is build and installed as a derivative into the nix store and added to the `PATH`. Running the following command will display all the available options :
+As part of nix setup, the `Packell` application is build and installed as a derivative into the nix
```

Comment:
```suggestion
As part of nix setup, the `Packell` application is built and installed as a derivative into the nix
```

Reply:
Also, now aren't we just downloading the binary?

Reply:
Not sure if this is still relevant given that the binary is currently downloaded

Reply:
It's still relevant as we have a different `nix shell` environment in `tools/packell` that's required for development. Once you enter this Packell directory, the `daml-finance` nix environment is unloaded and the `packell` environment is loaded with the GHC, Haskell Language Server, a Packell version built from source, etc. all available on the `PATH`.